### PR TITLE
Add `UKCC7` tournament article, add missing staff in `UKCC3`

### PR DESCRIPTION
**Pull request by Bazzadwarf**
Doing this one is slightly out of order, but since the file already existed from https://github.com/ppy/osu-wiki/pull/3266 I just went ahead and gave it a clean up to bring it in line with the current tournament wiki template and standards. Big thanks to @DMills94 for doing most of the busywork.

Also, added some staff members that was missing to the `UKCC3` article, whoops...

- [x] Host fact check.

### CODE REVIEW

**Comment by Bazzadwarf** [comment]
Ready for a review!

File: wiki/Tournaments/UKCC/7/en.md [hunk 1]
```diff
@@ -7,14 +7,685 @@ tags:
...
+- FreeMod
+  - [Byakko - Kachou Fuusetsu (EvilElvis) [MasterPeeysa]](https://osu.ppy.sh/beatmapsets/190603#osu/453790)
+  - [Denkishiki Karen Ongaku Shuudan - Owari Tsumugishi Mono (LMT) [Lasse's Extra]](https://osu.ppy.sh/beatmapsets/763607#osu/1711326)
+  - [Kucchy vs Akky - Satori \~3rd eyes\~ (DJPop) [Lunatic]](https://osu.ppy.sh/beatmapsets/13827#osu/50892)
+- Tiebreaker
+  - **[Demetori - Eastern Dream ~ The Introduction / Kamisabita Kosenjou \~ Suwa Foughten Field (Keada) [Stage 6]](https://osu.ppy.sh/beatmapsets/642547#osu/1362265)**
+
+### Semifinals
+
+**[Download the map pack here! (128.7MB)](https://mega.nz/#!S7p2QShR!ugelC74o3C1Xc3ECMHCb7EWzhvuZh75UuGs9_nznR44)**
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:149]:
```suggestion
**[Download the mappack here! (128.7MB)](https://mega.nz/#!S7p2QShR!ugelC74o3C1Xc3ECMHCb7EWzhvuZh75UuGs9_nznR44)**
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 2]
```diff
@@ -7,14 +7,685 @@ tags:
...
+- FreeMod
+  - [All That Remains - This Calling (Riince) [Despair]](https://osu.ppy.sh/beatmapsets/304439#osu/682053)
+  - [Wiklund - Billy Boogie (yeahyeahyeahhh) [16 Bits?!]](https://osu.ppy.sh/beatmapsets/9040#osu/36547)
+  - [Akiyama Uni - The Grimoire of Alice (Hollow Wings) [Extra]](https://osu.ppy.sh/beatmapsets/123839#osu/316018)
+- Tiebreaker
+  - **[USAO - Showdown (Depths) [Calamity]](https://osu.ppy.sh/beatmapsets/688191#osu/1456429)**
+
+### Quarterfinals
+
+**[Download the map pack here! (141.7MB)](https://mega.nz/#!37BHiSQa!_YCHsfzKe3AqxPboSBa2Uuyf0_chbWqYVTpd5QNj9WA)**
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:180]:
```suggestion
**[Download the mappack here! (141.7MB)](https://mega.nz/#!37BHiSQa!_YCHsfzKe3AqxPboSBa2Uuyf0_chbWqYVTpd5QNj9WA)**
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 3]
```diff
@@ -7,14 +7,685 @@ tags:
...
+- FreeMod
+  - [TAKA respect for J.S.B - Ubertreffen (kiddly) [Another]](https://osu.ppy.sh/beatmapsets/15740#osu/56830)
+  - [Mikami Shiori - Watashi, Shuyaku no Akaza Akari desu (Happy Euri Mix) (Arphimigon) [Akarimigon]](https://osu.ppy.sh/beatmapsets/404937#osu/880364)
+  - [T & Sugah x Lexurus - No More (Strategas) [Extra]](https://osu.ppy.sh/beatmapsets/471562#osu/1007896)
+- Tiebreaker
+  - **[Diao Ye Zong feat. Meramipop - Gensou Roman Kikou (is the limit.) (sahuang) [Flower Dance]](https://osu.ppy.sh/beatmapsets/719447#osu/1519282)**
+
+### Round of 16
+
+**[Download the map pack here! (163.5MB)](https://mega.nz/#!qqA2nCSa!zCQWppNn47REZtWCPLk9aVguEF9rGuO9h6BwHQGVqUk)**
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:210]:
```suggestion
**[Download the mappack here! (163.5MB)](https://mega.nz/#!qqA2nCSa!zCQWppNn47REZtWCPLk9aVguEF9rGuO9h6BwHQGVqUk)**
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 4]
```diff
@@ -7,14 +7,685 @@ tags:
...
+- FreeMod
+  - [Cranky - The Five (tsukamaete) [Another]](https://osu.ppy.sh/beatmapsets/18339#osu/64996)
+  - [Airi Totoki, Akane Hino, Aiko Takamori, Syoko Hoshi, Yuko Hori - Zettai Tokken Shuchou Shimasu! (Snow Note) [MASTER]](https://osu.ppy.sh/beatmapsets/738362#osu/1559184)
+  - [Rigel Theatre - Rinn Ding Dong (kwk) [moph's Big Dong Extra]](https://osu.ppy.sh/beatmapsets/804648#osu/1724396)
+- Tiebreaker
+  - **[Demetori - Youyoumu \~ The Woman Who Sold the Moon (Keada) [Phantasm]](https://osu.ppy.sh/beatmapsets/740641#osu/1562486)**
+
+### Round of 32
+
+**[Download the map pack here! (114.6MB)](https://mega.nz/#!OvQhmAiS!PT1qpuu5XGjnvX9Ra9w8flaEZsL_CCW47scekuARt7Y)**
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:240]:
```suggestion
**[Download the mappack here! (114.6MB)](https://mega.nz/#!OvQhmAiS!PT1qpuu5XGjnvX9Ra9w8flaEZsL_CCW47scekuARt7Y)**
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 5]
```diff
@@ -7,14 +7,685 @@ tags:
...
+- FreeMod
+  - [nano - Silence (Tari) [Insane]](https://osu.ppy.sh/beatmapsets/240160#osu/555116)
+  - [Streetlight Manifesto - A Moment of Violence (pishifat) [Extra]](https://osu.ppy.sh/beatmapsets/485376#osu/1035104)
+  - [onoken - Felys (Inferior) [DJPop's Insane]](https://osu.ppy.sh/beatmapsets/7013#osu/160795)
+- Tiebreaker
+  - **[FELT - Lies in Reality (Frostings) [Albatross]](https://osu.ppy.sh/beatmapsets/568845#osu/1205981)**
+
+### Qualifying round 2
+
+**[Download the map pack here! (86.8MB)](https://mega.nz/#!f3ASAA7T!umH3bN99cnuYe1ZFsOiXauUBFQRrW0n-_Zr4V8gE9ZA)**
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:270]:
```suggestion
**[Download the mappack here! (86.8MB)](https://mega.nz/#!f3ASAA7T!umH3bN99cnuYe1ZFsOiXauUBFQRrW0n-_Zr4V8gE9ZA)**
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 6]
```diff
@@ -7,14 +7,685 @@ tags:
...
+- HardRock
+  - [nano - No pain, No game (xsrsbsns) [Insane]](https://osu.ppy.sh/beatmapsets/60785#osu/185761)
+  - [iru1919 - Tenko (Seto Kousuke) [Insane]](https://osu.ppy.sh/beatmapsets/840826#osu/1759876)
+- DoubleTime
+  - [u's - START:DASH!! (scanter) [Insane]](https://osu.ppy.sh/beatmapsets/99340#osu/264252)
+  - [Hana - Gekka Jasmine (2014ver.) (PinkHeart) [Gyro's Light Insane]](https://osu.ppy.sh/beatmapsets/408745#osu/1035068)
+
+### Qualifying round 1
+
+**[Download the map pack here! (46.7MB)](https://mega.nz/#!T6RjDY7S!0PbquOystkS4v2-byiBC0b_KMEZA8lECFhMqjYKDm_A)**
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:288]:
```suggestion
**[Download the mappack here! (46.7MB)](https://mega.nz/#!T6RjDY7S!0PbquOystkS4v2-byiBC0b_KMEZA8lECFhMqjYKDm_A)**
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 7]
```diff
@@ -7,14 +7,685 @@ tags:
...
+- Hidden
+  - [Iguchi Yuka - RE-ILLUSION (Mordred) [Collab Insane]](https://osu.ppy.sh/beatmapsets/806635#osu/1856697)
+- HardRock
+  - [Meramipop - algorhythm (cRyo[iceeicee]) [Insane]](https://osu.ppy.sh/beatmapsets/43524#osu/136571)
+- DoubleTime
+  - [T.M.Revolution x Mizuki Nana - Preserved Roses (Xinely) [Collab Hard]](https://osu.ppy.sh/beatmapsets/223530#osu/522135)
+
+## Match schedules and results
+
+![Schedules](img/schedule_no_number.png)
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:304]:
```suggestion
## Match schedule

![Match schedule](img/schedule_no_number.png)

## Match results
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 8]
```diff
@@ -7,14 +7,685 @@ tags:
...
+| --: | :-: | :-: | :-- | :-- |
+| **Spare** ![][flag_GB] | **5** | 0 | ![][flag_GB] bloo | [#1](https://osu.ppy.sh/community/matches/55335898) |
+| **AJT** ![][flag_GB] | **5** | 3 | ![][flag_GB] Best459 | [#1](https://osu.ppy.sh/community/matches/55350496) |
+
+Sunday, 6 October 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Harpy** ![][flag_GB] | **5** | 3 | ![][flag_GB] Doomsday | [#1](https://osu.ppy.sh/community/matches/55372794) |
+| Wario ![][flag_GB] | 2 | 5 | ![][flag_GB] **Ventus** | [#1](https://osu.ppy.sh/community/matches/55378053) |
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:424]:
```suggestion
| Wario ![][flag_GB] | 2 | **5** | ![][flag_GB] **Ventus** | [#1](https://osu.ppy.sh/community/matches/55378053) |
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 9]
```diff
@@ -7,14 +7,685 @@ tags:
...
+| --: | :-: | :-: | :-- | :-- |
+| **Apo11o** ![][flag_GB] | **5** | 2 | ![][flag_GB] Defex | [#1](https://osu.ppy.sh/community/matches/55236437) |
+
+Saturday, 5 October 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Kingling** ![][flag_GB] | **0** | -1 | ![][flag_GB] meltz | *forfeit* |
+| MoeYandere ![][flag_GB] | -1 | **0** | ![][flag_GB] hlo | *forfeit* |
+| majoreh ![][flag_GB] | 2 | **5** | ![][flag_GB] Zyphox | [#1](https://osu.ppy.sh/community/matches/55347021) |
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:440]:
```suggestion
| MoeYandere ![][flag_GB] | -1 | **0** | ![][flag_GB] **hlo** | *forfeit* |
| majoreh ![][flag_GB] | 2 | **5** | ![][flag_GB] **Zyphox** | [#1](https://osu.ppy.sh/community/matches/55347021) |
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 10]
```diff
@@ -7,14 +7,685 @@ tags:
...
+
+Sunday, 6 October 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Malteser** ![][flag_GB] | **5** | 3 | ![][flag_GB] Woffel | [#1](https://osu.ppy.sh/community/matches/55368204) |
+| **Bubbleman** ![][flag_GB] | **5** | 1 | ![][flag_GB] Rinnu | [#1](https://osu.ppy.sh/community/matches/55371230) |
+| Voss ![][flag_GB] | 2 | 5 | ![][flag_GB] **Kerantina** | [#1](https://osu.ppy.sh/community/matches/55376109) |
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:448]:
```suggestion
| Voss ![][flag_GB] | 2 | **5** | ![][flag_GB] **Kerantina** | [#1](https://osu.ppy.sh/community/matches/55376109) |
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 11]
```diff
@@ -7,14 +7,685 @@ tags:
...
+| **Voss** ![][flag_GB] | **5** | 1 | ![][flag_GB] Woffel | [#1](https://osu.ppy.sh/community/matches/55040015) |
+| **Harpy** ![][flag_GB] | **5** | 0 | ![][flag_GB] Defex | [#1](https://osu.ppy.sh/community/matches/55042213) |
+| **helix** ![][flag_GB] | **5** | 1 | ![][flag_GB] Rinnu | [#1](https://osu.ppy.sh/community/matches/55043513) |
+| **Apo11o** ![][flag_GB] | **0** | -1 | ![][flag_GB] DeltaZero | *forfeit* |
+
+Monday, 23 September 2019:
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| -spoon- ![][flag_GB] | 4 | **5** | ![][flag_GB] Wario | [#1](https://osu.ppy.sh/community/matches/55061725) |
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:574]:
```suggestion
| -spoon- ![][flag_GB] | 4 | **5** | ![][flag_GB] **Wario** | [#1](https://osu.ppy.sh/community/matches/55061725) |
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 12]
```diff
@@ -7,14 +7,685 @@ tags:
...
+| --: | :-: | :-: | :-- | :-- |
+| **Ventus** ![][flag_GB] | **5** | 4 | ![][flag_GB] Summer Solstice | [#1](https://osu.ppy.sh/community/matches/55103447) |
+
+### Qualifying round 2
+
+See the [Qualifying round 2 results sheet](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit#gid=1213327055) for a breakdown of the round 2 qualification results.
+
+Top 10 qualification results:
+
+| Placing | Player | Total Score | Average Score |
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:588]:
```suggestion
| Placing | Player | Total score | Average score |
```

File: wiki/Tournaments/UKCC/7/en.md [hunk 13]
```diff
@@ -7,14 +7,685 @@ tags:
...
+| *9th place* | ![][flag_GB] [bloo](https://osu.ppy.sh/users/6778877) | 8,337,871 | 926,430 |
+| *10th place* | ![][flag_GB] [Apo11o](https://osu.ppy.sh/users/9558549) | 7,934,353 | 881,595 |
+
+### Qualifying round 1
+
+See the [Qualifying round 1 results sheet](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit#gid=1010206758) for a breakdown of the round 1 qualification results.
+
+Top 10 qualification results:
+
+| Placing | Player | Total Score | Average Score |
```
Comment (TPGPL) [inline wiki/Tournaments/UKCC/7/en.md:607]:
```suggestion
| Placing | Player | Total score | Average score |
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/UKCC/7/en.md (hunk 7 above)
Comment (Bazzadwarf) [reply wiki/Tournaments/UKCC/7/en.md:304]:
```suggestion
## Match results
```

**Review by TPGPL — APPROVED** [review]

**Pull request merged**